Helper/Data.php

```diff
@@ -171,6 +172,15 @@ public function getAuthResponse()
         return $this->authResponse;
     }
 
+    /**
+     * Pay on PagSeguro redirect method - check if should redirect to success page (true) or directly to PagSeguro (false)
+     * @return bool
+     */
+    public function isRedirectToSuccessPageEnabled()
+    {
+        return $this->scopeConfig->getValue(self::XML_PATH_PAYMENT_PAGSEGURO_REDIRECT_TO_SUCCESSPAGE, ScopeInterface::SCOPE_WEBSITE);
```

Comment:
In the system.xml you allow the admin to configure it in a store view level, but here the store view level is ignored by the SCOPE_WEBSITE.